src/System.Management.Automation/FormatAndOutput/common/PSStyle.cs

```diff
@@ -259,6 +271,27 @@ public string FromRgb(int rgb)
             }
         }
 
+        /// <summary>
+        /// Contains configuration for the progress bar visualization.
+        /// </summary>
+        public class ProgressConfiguration
+        {
+            /// <summary>
+            /// Gets or sets the style for progress bar.
+            /// </summary>
+            public string Style { get; set; } = "\x1b[33;1m";
+
+            /// <summary>
+            /// Gets or sets the max width of the progress bar.
+            /// </summary>
+            public int MaxWidth { get; set; } = 120;
+
+            /// <summary>
+            /// Gets or sets the style for progress bar.
+            /// </summary>
+            public ProgressView View { get; set; } = ProgressView.Minimal;
```

Comment:
Maybe ViewStyle if it is a _style_.

Reply:
Trying not to overload the `style` term and use it to mean ANSI decoration.  Will change the description to `view`.